start.sh

```diff
@@ -7,13 +7,16 @@ NIC=virtio
 
 if [ "$NIC" = "virtio" ]; then
     if [ ! -e virtio*.iso ]; then
-        echo Fetching virtIO drivers...
-        VIRTIO_ISO=$(wget -q -O - http://alt.fedoraproject.org/pub/alt/virtio-win/stable/ \
-                    | grep -oiE 'virtio-win-[0-9.\-]+iso' \
-                    | head -1)
-        wget http://alt.fedoraproject.org/pub/alt/virtio-win/stable/${VIRTIO_ISO}
+        # RH now have a package
+        if [ -e "/usr/share/virtio-win/virtio-win.iso" ]; then
+            echo "Using ISO from virtio-win package."
+            ln -s /usr/share/virtio-win/virtio-win.iso ./virtio-win.iso;
```

Comment:
This symlink seems messy, you could just set CDIMAGE to point straight at the ISO file, and combine this with the outer if statement.